dart/lib/src/hub_adapter.dart

```diff
@@ -77,4 +77,7 @@ class HubAdapter implements Hub {
 
   @override
   SentryId get lastEventId => Sentry.lastEventId;
+
+  @override
+  void addIntegrationDisposer(disposer) {}
```

Comment:
@marandaneto I'm not sure if this method must do something in the HubAdapter ?

Reply:
I think this should not be part of the Hub at all.
See https://github.com/getsentry/sentry-dart/pull/209#discussion_r533322127

Reply:
✅